lib/plugins/aws/package/compile/events/s3/configSchema.js

```diff
@@ -0,0 +1,813 @@
+'use strict';
+
+const AbortIncompleteMultipartUpload = {
```

Comment:
Let's use non-capitalized var names (in JS convention is to use capitalized var names only for function constructors - I'd love lint rule for that, still it could work reliably only in places as here, where we directly assign literals)